packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -478,14 +493,27 @@ export default [
       needUpdateParams: (state, { job, schedules }) =>
         job !== undefined && schedules !== undefined && !state.paramsUpdated,
       isJobInvalid: state =>
-        state.name.trim() === '' ||
-        (isEmpty(state.schedules) && isEmpty(state.newSchedules)) ||
-        (isEmpty(state.vms) && !state.smartMode) ||
-        ((state.backupMode || state.deltaMode) && isEmpty(state.remotes)) ||
-        ((state.drMode || state.crMode) && isEmpty(state.srs)) ||
-        (state.exportMode && !state.exportRetentionExists) ||
-        (state.snapshotMode && !state.snapshotRetentionExists) ||
-        (!state.isDelta && !state.isFull && !state.snapshotMode),
+        state.missingName ||
+        state.missingVms ||
+        state.missingBackupMode ||
+        state.missingSchedules ||
+        state.missingRemotes ||
+        state.missingSrs ||
+        state.missingExportRetention ||
+        state.missingSnapshotRetention,
+      missingName: state => state.name.trim() === '',
+      missingVms: state => isEmpty(state.vms) && !state.smartMode,
+      missingBackupMode: state =>
+        !state.isDelta && !state.isFull && !state.snapshotMode,
+      missingRemotes: state =>
+        (state.backupMode || state.deltaMode) && isEmpty(state.remotes),
+      missingSrs: state => (state.drMode || state.crMode) && isEmpty(state.srs),
+      missingSchedules: state =>
+        isEmpty(state.schedules) && isEmpty(state.newSchedules),
```

Comment:
Why `&& isEmpty(state.newSchedules)`? Do you take whatever is in `newSchedules` when the user clicks Create?